Xamarin.Forms.Build.Tasks/SetPropertiesVisitor.cs

```diff
@@ -283,7 +283,9 @@ internal static string GetContentProperty(TypeReference typeRef)
 				out markupExtension, out genericArguments))
 			{
 				var acceptEmptyServiceProvider = vardefref.VariableDefinition.VariableType.GetCustomAttribute(module, ("Xamarin.Forms.Core", "Xamarin.Forms.Xaml", "AcceptEmptyServiceProviderAttribute")) != null;
-				if (vardefref.VariableDefinition.VariableType.FullName == "Xamarin.Forms.Xaml.BindingExtension")
+				if (   vardefref.VariableDefinition.VariableType.FullName == "Xamarin.Forms.Xaml.BindingExtension"
```

Comment:
I noticed this file has many places where unnecessary whitespace is being inserted after the open parenthesis.

Reply:
that's on purpose, so conditions are indented like this
```
if (   a
    && b)
```
which is more readable (for long and/or complex conditions) than having the operator at the end of the line or at the beginning without indentation